### PR TITLE
chore: changelog 49.9.0

### PR DESCRIPTION
**Pull request by xyc**
### What does this PR do?
Changelog for release v49.9.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+
+## Fixed
+
+#### salesforcedx-vscode-lightning
+
+- chore: update to latest lwc/aura lang server version ([PR #2495](https://github.com/forcedotcom/salesforcedx-vscode/pull/2495))
+
+#### salesforcedx-vscode-lwc
+
+- chore: update to latest lwc/aura lang server version ([PR #2495](https://github.com/forcedotcom/salesforcedx-vscode/pull/2495))
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md:11]:
```suggestion
- Update to the latest version of the Aura and LWC language servers ([PR #2495](https://github.com/forcedotcom/salesforcedx-vscode/pull/2495))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
+# 49.9.0 - September 10, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-lightning
+
+- chore: update to latest lwc/aura lang server version ([PR #2495](https://github.com/forcedotcom/salesforcedx-vscode/pull/2495))
+
+#### salesforcedx-vscode-lwc
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md:9]:
```suggestion
#### salesforcedx-vscode-lightning, salesforcedx-vscode-lwc
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
+# 49.9.0 - September 10, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-lightning
+
+- chore: update to latest lwc/aura lang server version ([PR #2495](https://github.com/forcedotcom/salesforcedx-vscode/pull/2495))
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]: I would remove this section

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
+
+#### salesforcedx-vscode
+
+- chore: changelog 49.8.0 updates ([PR #2486](https://github.com/forcedotcom/salesforcedx-vscode/pull/2486))
+
+#### docs
+
+- Updates to Nate's [code completion](https://developer.salesforce.com/tools/vscode/en/apex/writing) documentation updates (minor edits) ([PR #2471](https://github.com/forcedotcom/salesforcedx-vscode/pull/2471))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:19]:
```suggestion
- Updates to [Code Completion](https://developer.salesforce.com/tools/vscode/en/apex/writing) documentation ([PR #2471](https://github.com/forcedotcom/salesforcedx-vscode/pull/2471))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,25 @@
...
+
+- Added [link to community extensions](https://developer.salesforce.com/tools/vscode/en/getting-started/recommended-extensions) ([PR #2477](https://github.com/forcedotcom/salesforcedx-vscode/pull/2477))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:21]:
```suggestion
- Added [Community Extensions](https://developer.salesforce.com/tools/vscode/en/getting-started/recommended-extensions/#community-extensions) section ([PR #2477](https://github.com/forcedotcom/salesforcedx-vscode/pull/2477))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,25 @@
+# 49.9.0 - September 10, 2020
+
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md:2]:
```suggestion
## Added

### salesforcedx-vscode-apex
- Added support for the new [Safe Navigator Operator](https://releasenotes.docs.salesforce.com/en-us/winter21/release-notes/rn_apex_SafeNavigationOperator.htm) within the Apex Language Server Protocol ([PR #2507](https://github.com/forcedotcom/salesforcedx-vscode/pull/2507))

```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,22 @@
+# 49.9.0 - September 10, 2020
+## Added
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md:2]:
```suggestion

## Added
```

**Review by sfsholden — APPROVED** [review]

**Review by lcampos — APPROVED** [review]

**Pull request merged**